### DISCUSSION TITLE
BMP388 - CJMCU-388 suported?

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#15176] [Q&A] [answered]
is or will be the sensor: 

`BMP388 - CJMCU-388`

supported??
Thx
Mark

**Comment by barbudor** [comment, marked as answer]
This pressure sensor is end-of-life
There are already 9 models of pressure sensors supported including better Bosch ones

**Reply by Mark-Muc** [reply]
OK Thanks .. i got still 2 of them .. btw. It's a BOSCH one

**Reply by sfromis** [reply]
Looks like the Bosch-suggested replacement BMP390 is not Tasmota-supported either. The support in Tasmota is mostly for the combined sensors BME280, BME680, BME688, and even the outdated BMP280, BMP180 and BMP085. Of course, anyone can submit a driver, but most people want to have at least temperature measured at the same time.
